db/config.ts

```diff
@@ -1,16 +1,16 @@
 import type { Config } from "drizzle-kit";
 export const dbCredentials = {
-    host: process.env.POSTGRES_HOST || "0.0.0.0",
-    port: parseInt(process.env.POSTGRES_PORT || '5432'),
-    user: process.env.POSTGRES_USER || "postgres",
-    password: process.env.POSTGRES_PASSWORD || "postgres",
-    database: process.env.POSTGRES_DB || "medium"
+    host: Bun.env.POSTGRES_HOST || "0.0.0.0",
+    port: parseInt(Bun.env.POSTGRES_PORT || '5432'),
+    user: Bun.env.POSTGRES_USER || "postgres",
+    password: Bun.env.POSTGRES_PASSWORD || "postgres",
+    database: Bun.env.POSTGRES_DB || "medium"
 }
 
 export const dbCredentialsString = `postgres://${dbCredentials.user}:${dbCredentials.password}@${dbCredentials.host}:${dbCredentials.port}/${dbCredentials.database}`;
 
 export default {
-    out: "./src/db/migrations",
+    out: "./migrations",
```

Comment:
```suggestion
    out: "./db/migrations",
```

The path is in relative to the root of the directory where the script is executed, so when it's executed from the root of the project the above would be correct.